graylog2-web-interface/src/views/components/Query.tsx

```diff
@@ -81,6 +87,27 @@ const _renderWidgetGrid = (widgetDefs, widgetMapping, results, positions, queryI
     }
   });
 
+  if (focusedWidget) {
```

Comment:
I think we should move this to a separate function itself (`renderFocussedWidget`?) and do a conditional in the `Query` function, e.g. `return focussedWidget ? renderFocussedWidget(...) : renderWidgetGrid(...);` - this makes it easy to understand.